scary_game.py

```diff
@@ -0,0 +1,293 @@
+import random
+import time
+import sys
+
+
+def buffer_print(phrase):
+    for i in phrase:
+        sys.stdout.flush()
+        print(i, end="")
+        time.sleep(.05)
+
+
+phrase_one = ("NARRATOR: The father and his daughter are on vacation in a log cabin in the forrest.\n"
+              "DADDY: Would you like to go on a late night walk and clear our mind?\n"
+              "SHEENA: yes DADDY, the weather is really nice too!\n"
+              "DADDY: Make sure you don't leave my side ok, and do not let go of my hand.\n"
+              "SHEENA: Yes DADDY, I will make sure to never leave your sight! Now can we go dad!\n"
+              "NARRATOR: They set off for a walk not knowing that this might be their last.\n"
+              "NARRATOR: On the walk they go. While on the walk the father looks behind to see if her daughter is there, \n"
+              "but what he sees terrifies him.\n"
+              "Monster: I have your daughter, and the only way to get her back, is to play my little game!\n")
+
+skip = input("Would you like to skip the cutscene? (y/n) \n=> ")
+if skip == "y":
+    pass
+elif skip == "n":
+    buffer_print(phrase_one)
+
+battery = "Battery"
+deagle = "Deagle"
+matches = "Matches"
+bullets = "Bullets"
+magazine = "Magazine"
+pistol = "Pistol"
+
+
+# This is where the actual game will run from
+class Game_engine(object):
+
+    # This is the constructor method, this is what will be passed to the play() for the game to run
+    def __init__(self, demon, item, father, ):
+        self.demon = demon
```

Comment:
instead of passing in `Demon()`, why don't you just initialize a `Demon()` here.